File: Conditional_Statements_Advanced_Exercise/operations_between_numbers.py
num1 = int(input())
num2 = int(input())
operator = input()

x = ""

if operator == "+":
    result = num1 + num2
    if result % 2 == 0:
        x = "even"
    else:
        x = "odd"
    print(f"{num1} {operator} {num2} = {result} - {x}")
elif operator == "-":
    result = num1 - num2
    if result % 2 == 0:
        x = "even"
    else:
        x = "odd"
    print(f"{num1} {operator} {num2} = {result} - {x}")
elif operator == "*":
    result = num1 * num2
    if result % 2 == 0:
        x = "even"
    else:
        x = "odd"
    print(f"{num1} {operator} {num2} = {result} - {x}")

if operator == "/":
    if num1 and num2 != 0:
        result = num1 / num2
        print(f"{num1} / {num2} = {result:.2f}")
    else:
        print(f"Cannot divide {num1} by zero")

if operator == "%":
    if num1 and num2 != 0:
        result = num1 % num2
        print(f"{num1} % {num2} = {result}")
    else:
        print(f"Cannot divide {num1} by zero")

File: C++/ConditionalStatements_Advanced_Exercise/OperationsBetweenNumbers/OperationsBetweenNumbers.cpp
#include <iostream>
using namespace std;

int main()
{
	int n1, n2;
	string operation;
	cin >> n1 >> n2 >> operation;

	double totalSumDouble = 0.0;
	int totalSumInt = 0;
	string oddOrEven;

	if (operation == "+") {
		totalSumInt = n1 + n2;
		if (totalSumInt % 2 == 0) {
			oddOrEven = "even";
		}
		else {
			oddOrEven = "odd";
		}
		cout << n1 << " " << operation << " " << n2 << " = " << totalSumInt << " - " << oddOrEven << endl;
	}
	else if (operation == "-") {
		totalSumInt = n1 - n2;
		if (totalSumInt % 2 == 0) {
			oddOrEven = "even";
		}
		else {
			oddOrEven = "odd";
		}
		cout << n1 << " " << operation << " " << n2 << " = " << totalSumInt << " - " << oddOrEven << endl;
	}
	else if (operation == "*") {
		totalSumInt = n1 * n2;
		if (totalSumInt % 2 == 0) {
			oddOrEven = "even";
		}
		else {
			oddOrEven = "odd";
		}
		cout << n1 << " " << operation << " " << n2 << " = " << totalSumInt << " - " << oddOrEven << endl;
	}
	else if (operation == "/" || operation == "%") {
		cout.setf(ios::fixed);
		cout.precision(2);

		if (n2 != 0) {
			if (operation == "/") {
				totalSumDouble = (double)n1 / n2;
				cout << n1 << " " << operation << " " << n2 << " = " << totalSumDouble << endl;
			}
			else if (operation == "%") {
				totalSumInt = n1 % n2;
				cout << n1 << " " << operation << " " << n2 << " = " << totalSumInt << endl;
			}
		}
		else {
			cout << "Cannot divide " << n1 << " by zero" << endl;

			return 0;
		}
	}
}